pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -613,8 +608,8 @@ func (configgen *ConfigGeneratorImpl) buildSidecarInboundListeners(
 	return listeners
 }
 
-func (configgen *ConfigGeneratorImpl) buildSidecarInboundHTTPListenerOptsForPortOrUDS(node *model.Proxy, pluginParams *plugin.InputParams) *httpListenerOpts {
-	clusterName := pluginParams.InboundClusterName
```

Comment:
why are you removing this line?

Reply:
instead it's passed into the function because plugin params no longer contains it